source/python/idetect/fact_extractor.py

```diff
@@ -2,17 +2,55 @@
 
 How to ensure has access to pre-loaded models?
 '''
-from interpreter import Interpreter, person_reporting_terms, structure_reporting_terms, person_reporting_units, \
+from idetect.interpreter import Interpreter, person_reporting_terms, structure_reporting_terms, person_reporting_units, \
     structure_reporting_units, relevant_article_terms
 import spacy
+from sqlalchemy.orm import object_session
+from idetect.model import Report, Location, Country
+from idetect.geotagger import get_geo_info
+import json
 
-nlp = spacy.load("en")
+nlp = spacy.load("en_default")
 print("Loaded Spacy English Language NLP Models.")
 
 
-def extract_facts(content):
+def extract_reports(article):
     # Get rules-based facts along with sentence numbers
     interpreter = Interpreter(nlp, person_reporting_terms, structure_reporting_terms, person_reporting_units,
                               structure_reporting_units, relevant_article_terms)
+    session = object_session(article)
+    content = article.content[0].content
```

Comment:
I think so....given that we're doing versioned articles, if we scrape again then we create a new copy of the article, so there is no reason for an article to have more than one piece of content.